collectors/python.d.plugin/postgres/postgres.chart.py

```diff
@@ -1332,3 +1358,35 @@ def create_lines(slot, lines):
     definitions[chart_name] = {
         'options': [name, title + ': ' + slot_name, units, 'replication slot files', context, chart_type],
         'lines': create_lines(slot_name, chart_template['lines'])}
+
+def add_stat_replications_chart(s,order, definitions, name, secondaries):
+    def create_lines(lines, secondaries):
+        result = list()
+        for line in lines:
+            for secondary in secondaries:
+                secondary=secondary.replace("-","")
+                metric='_'.join([secondary, line[0]])  
+                METRICS[QUERY_NAME_STAT_REPLICATION].append(metric)
```

Comment:
`METRICS` is a global parameter. We shouldn't rewrite it. It could affect other data collection jobs.

In addition, we modify the parameter in the `add_stat_replications_chart` function - its name doesn't reflect the logic (i expect the function only add charts).

Reply:
just add graphic